### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by twigington** [jonspalding/lisp-paredit#5]
I'm running Atom 1.0.7 and when I try to install lisp-paredit I get the following installation error.

```
npm WARN `git config --get remote.origin.url` returned wrong result (git://github.com/jonspalding/paredit.js) 
npm ERR! git clone git://github.com/jonspalding/paredit.js Cloning into bare repository 'C:\Users\twigington\.atom\.apm\_git-remotes\git-github-com-jonspalding-paredit-js-a323f904'...
npm ERR! git clone git://github.com/jonspalding/paredit.js fatal: read error: Invalid argument
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Users\\twigington\\AppData\\Local\\atom\\app-1.0.7\\resources\\app\\apm\\bin\\\\node.exe" "C:\\Users\\twigington\\AppData\\Local\\atom\\app-1.0.7\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\twigington\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\twigington\\.atom\\.apmrc" "install" "C:\\Users\\TWIGIN~1\\AppData\\Local\\Temp\\d-115713-8740-x8o9wa\\package.tgz" "--target=0.22.0" "--arch=ia32" "--msvs_version=2010"
npm ERR! node v0.10.35
npm ERR! npm  v2.5.1
npm ERR! code 128

npm ERR! Command failed: Cloning into bare repository 'C:\Users\twigington\.atom\.apm\_git-remotes\git-github-com-jonspalding-paredit-js-a323f904'...
npm ERR! fatal: read error: Invalid argument
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <http://github.com/npm/npm/issues>
```

**Comment by jonspalding** [comment]
Thanks for reporting, I've not tested on Windows yet. I'll get an environment set up and try to reproduce.

**Comment by jonspalding** [comment]
It worked for me on Windows 10.

It may be an issue with using a git reference in the package.json without a specific commit. I changed this in 0.4.2 so you could try reinstalling.

**Comment by twigington** [comment]
I just tried installing with 0.4.2 and getting the same error.

Thanks for looking into this.

**Comment by jonspalding** [comment]
I've just released 0.4.3 which now removes the git dependency in favour of a standard npm reference. This should fix it although some Clojure indentation will be incorrect until a fix is made in the upstream paredit.js.

Can you try again and let me know if successful?

**Comment by twigington** [comment]
That was successful. Thanks.
